Python/optimizer_bytecodes.c

```diff
@@ -437,6 +443,12 @@ dummy_func(void) {
         }
     }
 
+    op(_LOAD_ATTR, (owner -- attr, self_or_null if (oparg & 1))) {
+        (void)owner;
```

Comment:
I'd rename `owner` to `unused` so you don't need the `(void)` thing:
```suggestion
    op(_LOAD_ATTR, (unused -- attr, self_or_null if (oparg & 1))) {
```

OTOH You could add `sym_set_non_null(owner);` instead?